update_models.py

```diff
@@ -0,0 +1,47 @@
+""" test models in the distribution
```

Comment:
This file seems to be a copy of `test_models.py` (including the changes in this PR to `test_models.py`). Was your plan to have different files for different functions (i.e., testing vs updating), or could one file be removed?

Reply:
@nickbianco my plan was to have both as we'll frequently be updating models, so that was for convenience. In reality if the script does both we may do away with the test_models script altogether.

Reply:
Or we can rename to process_models and have it take cmd line arguments to decide

Reply:
Either is fine with me. It just looked like the files were exactly the same to me, but maybe I missed something.

Reply:
You're absolutely right, I mistakenly committed the modified file before renaming. Now I restored the original. Thanks for the catch 👍

Reply:
Great. Perhaps you could update the comment at the top of this file to describe what is does. Then this PR should be good to go.